modin/engines/base/frame/axis_partition.py

```diff
@@ -95,33 +95,33 @@ def shuffle(self, func, lengths, **kwargs):
     def _wrap_partitions(self, partitions):
         return [self.partition_type(obj) for obj in partitions]
 
-    def coalesce(self, bind_ip=False):
+    def coalesce(self, get_ip=False):
```

Comment:
Do you think `force_materialize` is a better name for this? `coalesce` generally happens anyway.

Reply:
I was thinking on this. I would prefer something of these: `use_ip`, `materialize_ip`, or `get_ip` at last :) because ObjectRefs to data will not be materilized yet.

Reply:
In the case of `axis_partition`, we return a single object which will have to be materialized. `force_materialize` provides a future, so I think it still makes sense instead of `coalesce`.

Reply:
Do you want to rename `coalesce` to `force_materialization`? If so, I don't follow your thought. Why do we want to materialize `axis_partition` after its coaliscing? We need to return ObjectRef to the `axis_partitions` with saving asynchrony when `get_ip=False`.

Reply:
My thinking was: the assumption from the developers is that they can treat the axis partition as if it were already coalesced, so a `coalesce` api might cause confusion. `force_materialization` is a scarier name that will not encourage misuse. The fact that it is asynchronous does not change that the whole axis partition is materialized.

Reply:
Okay, I see. We probably need to rename it to `force_materilization`.